rust/impl/publisher/tools/path.rs

```diff
@@ -0,0 +1,14 @@
+/// Internal namespace.
+pub( crate ) mod private
+{
+  /// Check if path has a glob.
+  pub fn glob_is( path : &str ) -> bool
+  {
+    path.chars().any( | char | char == '*' || char == '?' || char == '[' )
```

Comment:
This realization covers trivial cases. Please, extend the realization. 

Also, see the method `.contains` in the `String`.